02 - Web UI Template/CognitiveSearch.UI/Search/SearchSchema.cs

```diff
@@ -70,13 +72,13 @@ public static SearchField ToSearchField(this Field field)
                 IsFacetable = field.IsFacetable ?? false,
                 IsFilterable = field.IsFilterable ?? false,
                 IsKey = field.IsKey ?? false,
-                IsRetrievable = field.IsRetrievable ?? false,
+                IsHidden = field.IsHidden ?? false,
                 IsSearchable = field.IsSearchable ?? false,
                 IsSortable = field.IsSortable ?? false
             };
         }
 
-        public static SearchSchema AddFields(this SearchSchema schema, IEnumerable<Field> fields)
+        public static SearchSchema AddFields(this SearchSchema schema, IList<Azure.Search.Documents.Indexes.Models.SearchField> fields)
```

Comment:
nit: just use `SearchField` and add a `using` statement

Reply:
Already existing a defining  SearchField in SearchSchema.cs，So do this